File: 3194-find-words-containing-character/find-words-containing-character.js
/**
 * @param {string[]} words
 * @param {character} x
 * @return {number[]}
 */
var findWordsContaining = function(arr, x) {

    let res = [];

    for(let i=0;i<arr.length;i++) {

        let singleWord = arr[i];

        for(let j=0;j<singleWord.length;j++)

        if(singleWord[j]==x) {
            res.push(i);
            break;
        }
    }
    return res;
    
};

File: 3194-find-words-containing-character/find-words-containing-character.java
class Solution {
    public List<Integer> findWordsContaining(String[] arr, char x) {

        List<Integer> res = new ArrayList<>();

        for(int i=0;i<arr.length;i++) {
            String singleWord = arr[i];

            for(int j=0;j<singleWord.length();j++) {

                if(singleWord.charAt(j)==x) {
                    res.add(i);
                    break;
                }
            }
        }

        return res;
        
    }
}